server.js

```diff
@@ -1,15 +1,62 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const cors = require('cors');
+// const requestPromise = require('request-promise');
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 7070;
+
+// bundles for each react component
+// const bundles = require('../public/services')
+const clientBundles = './public/services';
+// bundles for each servers
+const serverBundles = './templates/services';
+// local network address for each service
+const serviceConfig = require('./service-config.json');
+// fetch each bundle files and write new files into local directory
+const services = require('./loader.js')(clientBundles, serverBundles, serviceConfig);
+
+const React = require('react');
+const ReactDom = require('react-dom/server');
+const template = require('./htmlRender.js');
 
 
 app.use(morgan('dev'));
 app.use(cors());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static('public'));
+
+
+const renderComponents = (components, props = {}) => Object.keys(components).map((item) => {
```

Comment:
Having one arrow function with brackets inside another arrow function on the same line without brackets is confusing style.